src/Tribe/Template.php

```diff
@@ -172,7 +172,33 @@ public function set_template_context_extract( $value = false ) {
 	}
 
 	/**
-	 * Sets a Index inside of the global or local context
+	 * Gets the local template context
+	 *
+	 * @since  TBD
+	 *
+	 * @todo  Apply filters here for local context
+	 *
+	 * @return mixed
+	 */
+	public function get_template_context() {
+		return $this->context;
+	}
+
+	/**
+	 * Gets the global template context
+	 *
+	 * @since  TBD
+	 *
+	 * @todo  Apply filters here for global context
```

Comment:
@bordoni Do you need filters added?

Reply:
Not now, but I wanted to make sure we could do that for template filtering of local and global context from customers.